types/src/crypto/asymmetric_key.rs

```diff
@@ -48,9 +47,9 @@ const ED25519: &str = "Ed25519";
 pub const SECP256K1_TAG: u8 = 2;
 const SECP256K1: &str = "Secp256k1";
 
-const SECP256K1_SECRET_KEY_LENGTH: usize = 32;
-const SECP256K1_COMPRESSED_PUBLIC_KEY_LENGTH: usize = 33;
-const SECP256K1_SIGNATURE_LENGTH: usize = 64;
+const SECP256K1_SECRET_KEY_LENGTH: usize = secp256k1::util::SECRET_KEY_SIZE;
+const SECP256K1_COMPRESSED_PUBLIC_KEY_LENGTH: usize = secp256k1::util::COMPRESSED_PUBLIC_KEY_SIZE;
+const SECP256K1_SIGNATURE_LENGTH: usize = secp256k1::util::SIGNATURE_SIZE;
```

Comment:
We should import these secp256k1 consts.

Reply:
Fixed in aa7803f07b9aaa33d7c0d78c2dd138a4f5f6879e